src/main/java/com/lob/protocol/response/CheckResponse.java

```diff
@@ -95,8 +101,16 @@ public String getUrl() {
         return url;
     }
 
-    public TrackingResponse getTracking() {
-        return tracking;
+    public String getCarrier() {
+        return carrier;
+    }
+
+    public String getTrackingNumber() {
+        return trackingNumber;
+    }
+
+    public List<TrackingEventResponse> getTrackingEvents() {
+        return trackingEvents;
```

Comment:
Ah yeah, it looks like it's just some inconsistencies. If you could update those as well, that would be great!